sync-mixins.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+TMPDIR="./tmp"
+MIXIN_REPO="git@github.com:giantswarm/giantswarm-kubernetes-mixin.git"
+# clone a branch if provided
+BRANCH=${1:-""}
+
+# make a temporary dir to work in
+mkdir ${TMPDIR}
+
+if [ -z "${BRANCH}" ]; then
+    # clone the mixins repo
+    echo -e "\nCloning master branch:\n"
+    git clone ${MIXIN_REPO} ${TMPDIR}/mixins
+else
+    # clone the mixins repo branch
+    echo -e "\nCloning branch '${BRANCH}':\n"
+    git clone -b ${BRANCH} --single-branch ${MIXIN_REPO} ${TMPDIR}/mixins
+fi
+
+# get the current commit of the mixin repo
+cd ${TMPDIR}/mixins
+MIXIN_VER=$(git rev-parse HEAD)
+cd - > /dev/null
+
+# copy over the dashboards to the helm chart
+cp -a ${TMPDIR}/mixins/files/dashboards/* helm/dashboards/dashboards/mixin/
```

Comment:
Does this script meant to run manually or in CI

Reply:
It needs to be run manually but it could be done through CI once a month

Reply:
@mcbenjemaa this implies we need to split the kube-prometheus-stack mixins or add that into the giantswarm-kubernetes-mixin repo as well